### ISSUE TITLE
button chinese and number , english vertical align

### ISSUE THREAD

**Issue by kudorori** [GeekyAnts/NativeBase#1827]
## react-native, react and native-base version
```
"native-base": "^2.3.10",
"react": "16.3.1",
"react-native": "0.55.3"
```
## Steps to reproduce (code snippet or screenshot)
```
import React from "react";
import { Button as RNButton } from "react-native"
import { Container, Header, Left, Body, Right, Button, Icon, Title, Text } from 'native-base';

export default class extends React.Component {
  render() {
    return (
      <Container>
        <Button smail>
          <Icon name="settings"></Icon>
          <Text>2018年 4月</Text>
        </Button>
        <Button smail>
          <Icon name="settings"></Icon>
          <Text>中文 Eng 123</Text>
        </Button>
        <RNButton title="2018年4月"></RNButton>
        <Text>2018年 4月</Text>
      </Container>
    )
  }
}

```
## Screenshot of emulator/device
[img](https://imgur.com/9U6IYAu)

**Comment by SupriyaKalghatgi** [comment]
@kudorori Provide sufficient information for us to help

**Comment by kudorori** [comment]
@SupriyaKalghatgi  I filled in my code and hope to solve this problem

**Comment by SupriyaKalghatgi** [comment]
@kudorori Can you elaborate on your issue description? 

> button chinese and number , english vertical align

**Comment by kudorori** [comment]
@SupriyaKalghatgi  I really don't know how to describe this problem in words...
[https://imgur.com/QnimM9c](https://imgur.com/QnimM9c)

**Comment by SupriyaKalghatgi** [comment]
@kudorori https://imgur.com/QnimM9c this gives better explanation

**Comment by akhil-geekyants** [comment]
@kudorori As a workaround set lineHeight of Text to undefined.
```
<Text style={{ lineHeight: undefined}}>2018年 4月</Text>
```
This will be fixed in the next release.

**Comment by kudorori** [comment]
@akhil-geekyants  thanks!

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.3](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.3)
